tests/testapp/settings.py

```diff
@@ -140,14 +140,26 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/stable/howto/static-files/
 
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    "staticfiles": {
+        "BACKEND": os.environ.get(
+            "STATICFILES_STORAGE",
+            "django.contrib.staticfiles.storage.StaticFilesStorage",
```

Comment:
**question:** Is there a particular reason to make this an env var? Why not always use `ManifestStaticFilesStorage`?

Reply:
Testing with `ManifestStaticFilesStorage` requires that you've done `collectstatic` beforehand, for example:

```
STATICFILES_STORAGE=django.contrib.staticfiles.storage.ManifestStaticFilesStorage pytest
```

Fails with:

```
ValueError: Missing staticfiles manifest entry for 'wagtailadmin/css/core.css'
```

As the static files aren't in place. But if you do:

```
STATICFILES_STORAGE=django.contrib.staticfiles.storage.ManifestStaticFilesStorage ./testmanage.py collectstatic --no-input
STATICFILES_STORAGE=django.contrib.staticfiles.storage.ManifestStaticFilesStorage pytest
```

It works as expected - the static files are in place.

I've added the additional commands in tox.ini that will run collectstatic beforehand if you're running inside tox. If you're just doing quick development outside of tox - then it defaults to `StaticFilesStorage` - which is faster/easier if you need a quick feedback loop.

For most of our projects at DEV we try and have some form of CI that runs with `ManifestStaticFilesStorage`, as that's what we deploy with - so running CI with the same storage helps reduce any errors.

It's possibly a bit opinionated, but I've tried to follow something roughly along the lines of: https://github.com/wagtail/wagtail/blob/main/wagtail/test/settings.py#L68.

Reply:
Thanks for the explanation!